Modules/_sqlite/row.c

```diff
@@ -204,6 +204,84 @@ static PyObject* pysqlite_row_richcompare(pysqlite_Row *self, PyObject *_other,
     Py_RETURN_NOTIMPLEMENTED;
 }
 
+static PyObject *
+pysqlite_row_repr(pysqlite_Row *self)
+{
+    _PyUnicodeWriter writer;
+    int result;
+    Py_ssize_t nitems, i;
+    PyObject *start;
+    PyObject *name;
+    PyObject *value;
+
+    _PyUnicodeWriter_Init(&writer);
+
+    start = PyUnicode_FromFormat("<%s object; {", Py_TYPE(self)->tp_name);
+    if (start == NULL) {
+        goto error;
+    }
+    result = _PyUnicodeWriter_WriteStr(&writer, start);
+    Py_DECREF(start);
+    if (result < 0) {
```

Comment:
`result == -1`?

Reply:
yep, `_PyUnicodeWriter_WriteStr` can return `-1` when the unicode is not ready.

Reply:
Even though there is no difference in behavior (conventionally int functions are supposed to return -1 on errors), I've got the impression that "< 0" style is substantially more common than "== -1" one in the codebase.